modules/migration-manually-rolling-back-migration.adoc

```diff
@@ -4,12 +4,75 @@
 // * migration/migrating_4_2_4/troubleshooting-4-2-4.adoc
 
 [id='migration-manually-rolling-back-migration_{context}']
-= Manually rolling back a migration
+= Rolling back a migration
```

Comment:
There are multiple procedures within this module. The content needs to be modularized and each procedure should be its own module. And I would update file titles accordingly for each of those modules for SEO and user scan-ability in the TOC. So for example, "Rolling back a migration from the web console" and "Rolling back a migration from the CLI". 

Essentially, "Rolling back a migration" would be your assembly and the first two sentences in this file can remain in that assembly as the abstract. Then nest all the procedural modules for each specific rollback method within that assembly. 

You can then add your conditionals for each procedural module in the assembly.